re: remove deprecated solana event hooks and lines from web3walletprovider

snowwhitedev opened the pull request:
This PR is related to ticket https://smarttokenlabs.atlassian.net/browse/TKS-1047
I removed deprecated solana event hooks and some other few lines

nicktaras commented:
@snowwhitedev thanks for these changes. I've shared a ticket that I feel must be addressed before this task can be merged.
Which fixes a change that has impacted our code with regards to 'accounts-changed' hook.

Once resolved, let's re-review this ticket.

snowwhitedev commented:
> 

Thanks, I will look into TKS-1047

snowwhitedev commented:
@nicktaras  
I committed again
- Rebased with staging
- Split solana event hooks from registerNewAddress function to prevent recursive event registeration
